applications/matter_bridge/src/app_task.cpp

```diff
@@ -131,6 +136,29 @@ CHIP_ERROR AppTask::RestoreBridgedDevices()
 	return CHIP_NO_ERROR;
 }
 
+/* onoff plug part*/
+void AppTask::SmartplugOnOffEventHandler()
```

Comment:
Generally, we do not want to compile the parts of code related to smart plug for a configuration that does not support smart plug (the default one). I would suggest creating some config option that will be used to enable smart plug functionality and put it in the bridge's application Kconfig. Then you can enable this config in your snippet .conf file. And then you can use this config in preprocessor checks before every part of the cpp code related to smart plug.